### PR TITLE
rj/release prep 0.7

### PR DESCRIPTION
**Pull request by jordens**
Various release preparatory items.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,19 +9,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+### Fixed
+
+## [v0.7.0] - 2022-08-09
+
+### Added
+
 * Optional specification of a static IP address ([#509](https://github.com/quartiq/stabilizer/pull/509))
 * Temperature sensor measurement for Pounder and Stabilizer's CPU has been added [#575](https://github.com/quartiq/stabilizer/pull/575)
+* External clock support for Pounder
+* Auxiliary ADC support for Pounder
```
Comment (ryan-summers) [inline CHANGELOG.md:25]:
```suggestion
* Auxiliary ADC support for Pounder
* Added support for STATIC_IP configuration
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,19 +9,32 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
-
 * Minimum supported Rust version bumped to 1.62
+* The `pounder_v1_1` feature has been replaced by the `pounder_v1_0` feature with inverse meaning.
+* The `Telemetry` python client has been integrated into the `stabilizer` Python library
 
 ### Fixed
 
 * Fixed panics when using a batch size of 1 ([#540](https://github.com/quartiq/stabilizer/issues/540))
 * Fixed an issue where startup delays were 1/4 of what they should be ([#524](https://github.com/quartiq/stabilizer/issues/524))
+* Fixed Pounder GPIO extender issues by rewriting the `mcp280xx` crate
```
Comment (ryan-summers) [inline CHANGELOG.md:37]:
```suggestion
* Fixed Pounder GPIO extender issues by rewriting the `mcp280xx` crate
* Fixed improper calculation of the signal generator phase increment ([#543](https://github.com/quartiq/stabilizer/pull/543))
```

File: CHANGELOG.md (hunk 1 above)
Comment (jordens) [reply CHANGELOG.md:25]: That up there two lines above already, no?

Comment (ryan-summers) [reply CHANGELOG.md:25]: Woops, correct.

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,19 +9,33 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+### Removed
+
+### Changed
+
+### Fixed
+
+## [v0.7.0] - 2022-08-09
```
Comment (ryan-summers) [inline CHANGELOG.md:18]:
```suggestion
## [v0.7.0] - 2022-08-10
```

**Review by ryan-summers — APPROVED** [review]

**Pull request merged**